src/m1_facilitator/repositories/TransactionRepository.ts

```diff
@@ -56,13 +56,15 @@ class TransactionModel extends Model {
 export default class TransactionRepository extends Subject<Transaction> {
   private mutex: Mutex;
 
+  private modelInstance: any;
```

Comment:
The only thing we want to avoid here is `any`. Please check sequlize doc and necessary types.

Reply:
@sarvesh-ost , unable to determine the exact type and work around for it.
I think we can create the ticket for it.
Let me know what do you think.